packages/genome-nexus-ts-api-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "genome-nexus-ts-api-client",
   "description": "Genome Nexus API Client for TypeScript",
-  "version": "1.1.2-beta.0",
+  "version": "1.1.3-beta.0",
```

Comment:
We shouldn't update this package version, because there is no change in this API client. `yarn updatePackageVersion` incorrectly prompts this package to update because of the limitation in `lerna version`. It always checks against the latest tag on master, not the latest commit.